kfdef/kfctl_aws.v1.0.2.yaml

```diff
@@ -209,14 +209,6 @@ spec:
         name: manifests
         path: kfserving/kfserving-install
     name: kfserving-install
-  - kustomizeConfig:
```

Comment:
I am not sure if we want to change pinned manifest? 
1. This is not used actually, user get manifest from tag. 
2. It only matters if we plan to cherry-pick some changes to v1.0-branch. What's the development flow? @jlewi

Reply:
I agree, we shouldn't change pinned manifests.

Reply:
ok I reverted changes for pinned manifests